### PR TITLE
feat: dapp icon

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes BX-####
Figma link (if any):

## What changed (plus any additional context for devs)

introducing DappIcon component to fallback to this symbol https://www.figma.com/file/a33TH3fYEWNC6xTePxZ8sN/BX-Signing?node-id=251%3A21606&mode=dev in case we don't have the dapp icon

## Screen recordings / screenshots

https://www.loom.com/share/ce6e8a11664c400c9a3c816f57c575bf

### CODE REVIEW

File: src/entries/popup/components/AppConnection/AppConnectionNudgeSheet.tsx [hunk 1]
```diff
@@ -81,7 +81,7 @@ export const AppConnectionNudgeSheet = ({
                   borderRadius: '8px',
                 }}
               >
-                <Box
+                {/* <Box
```
Comment (greg-schrammel) [inline src/entries/popup/components/AppConnection/AppConnectionNudgeSheet.tsx:84]: comment?

File: src/entries/popup/components/DappIcon/DappIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+};
+
+const RADIUS_SIZE = {
+  '60px': '18px' as BoxStyles['borderRadius'],
+  '36px': '10px' as BoxStyles['borderRadius'],
+  '32px': '9px' as BoxStyles['borderRadius'],
+  '18px': '6px' as BoxStyles['borderRadius'],
+  '16px': '4px' as BoxStyles['borderRadius'],
+  '14px': '4px' as BoxStyles['borderRadius'],
+};
```
Comment (greg-schrammel) [inline src/entries/popup/components/DappIcon/DappIcon.tsx:24]: I think `satisfies Record<string, BoxStyles['borderRadius']>` would work too, so we don't need to `as BoxStyles['borderRadius']` it all

**Review by greg-schrammel — APPROVED** [review]

**Review by brunobar79 — APPROVED** [review]

File: src/entries/popup/components/DappIcon/DappIcon.tsx (hunk 2 above)
Comment (estebanmino) [reply src/entries/popup/components/DappIcon/DappIcon.tsx:24]: nice, i added 

```
Record<DappIconSize, BoxStyles['borderRadius']>
```

File: src/entries/popup/components/AppConnection/AppConnectionNudgeSheet.tsx (hunk 1 above)
Comment (estebanmino) [reply src/entries/popup/components/AppConnection/AppConnectionNudgeSheet.tsx:84]: oh will revert this

Comment (estebanmino) [reply src/entries/popup/components/AppConnection/AppConnectionNudgeSheet.tsx:84]: ah no, will remove this

**Pull request merged**